dffml/util/cli/cmd.py

```diff
@@ -87,21 +89,46 @@ def add_subs(self, add_from: "CMD"):
                 except argparse.ArgumentError as error:
                     raise Exception(repr(add_from)) from error
 
+        position_list = {}
+        for field in dataclasses.fields(add_from.CONFIG):
+            arg = mkarg(field)
+            if isinstance(arg, Arg):
+                if isinstance(field.metadata.get("position"), int):
+                    position_list[field.metadata.get("position")] = (
+                        field.name,
+                        arg,
+                    )
+                else:
+                    try:
+                        if position_list:
+                            for position in sorted(position_list.keys()):
+                                name, positional_arg = position_list[position]
+                                self.add_argument(name, **positional_arg)
+                            position_list.clear()
+                        if not position_list:
+                            self.add_argument("-" + field.name, **arg)
```

Comment:
We'll want to do `field.name.replace("_", "-")` whenever we add_argument for a field. So that things like, `double_arg` become `-double-arg`